### PR TITLE
WIP: Using ORCID sandbox by default

### PR DESCRIPTION
**Pull request by damnhipster**
#### Background

This allows us to continue using ORCID's sandbox for the demo environment.

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -27,6 +27,11 @@ module.exports = {
     from: 'dev@example.com',
     path: `${__dirname}/mailer`,
   },
+  'auth-orcid': {
+    clientID: '',
+    clientSecret: '',
+    sandbox: true,
```
Comment (giorgiosironi) [inline config/default.js]: I'd prefer to have the possibility to configure any URL, as in
https://github.com/elifesciences/profiles-formula/blob/master/salt/pillar/profiles.sls#L11
where we can configure the real ORCID, the sandbox ORCID, or a [dummy](https://github.com/elifesciences/orcid-dummy). `https://orcid.org/` is probably fine-grained enough

Comment (tamlyn) [reply config/default.js]: Good point. But bear in mind the whole ORCID login functionality is being removed imminently, see #636

Comment (diversemix) [reply config/default.js]: @giorgiosironi ... Is this okay then as its just for the short-term?

File: config/default.js [hunk 2]
```diff
@@ -27,6 +27,11 @@ module.exports = {
     from: 'dev@example.com',
     path: `${__dirname}/mailer`,
   },
+  'auth-orcid': {
+    clientID: '',
+    clientSecret: '',
```
[the remark this answers is not in the document]
Comment (diversemix) [reply config/default.js]: The way `default.js` is used is more like a template... the values are overridden in the other config files such as `local.js` and `development.js`

**Comment by diversemix** [comment]
"Work in Progress" ... i.e. there is probably more to come, or don't merge just yet. 

I approved it as I was responding to an email that had the title without the "WIP"  :/

**Comment by damnhipster** [comment]
Addressed by https://github.com/elifesciences/elife-xpub-deployment/pull/10/files#diff-4e5e90c6228fd48698d074241c2ba760R22

**Pull request closed without merging**